resources/stylesheet.css

```
@@ -28,6 +28,32 @@ html.darkMode {
 	--tertiaryHover: #27292c;
 }
 
+html.terminalMode {
+    --primary: #121212;
+    --secondary: #333;
+    --secondaryHover: #555;
+    --textPrimary: #32cd32;
+    --textSecondary: #90f;
+    --linkHover: #a1c6ff;
+    --searchBar: #222b2e;
+    --saturation: 60%;
+    --tertiary: #2a2b2d;
+    --tertiaryHover: #565a60;
+}
+
+html.scratchMode {
+    --primary: #fcfcfc;
+    --secondary: #4d97fe;
+    --secondaryHover: #5686de;
+    --textPrimary: #565e75;
+    --textSecondary: #ffffff;
+    --linkHover: #4180d7;
+    --searchBar: #ffffff;
+    --saturation: 100%;
+    --tertiary: #4280D9;
+    --tertiaryhover: #2d65b3;
```

You should use an uppercase H for the variable :P

```suggestion
    --tertiaryHover: #2d65b3;
```